_data/accessibility-tests/tooltip.yml

```diff
@@ -0,0 +1,86 @@
+title: Tooltip
+component_status: pass
+test_items:
+# General tests
+  - summary: Color is not the only method used to communicate the purpose of the tool tip.
+    summary_additional: |
+      When you view the tooltip, the text, and never color alone, convey its purpose.
+    test_status: pass
+    test_type: general
+    version_tested: 3.8.0
+    wcag_criterion: 1.4.1
+  - summary: Appropriate color contrast between tooltip text and background.
+    summary_additional: |
+      When you view the tooltip and use ANDI or color contrast analyzer to look at the hex colors,
+      contrast between the background and text is at least 4:5:1.
+    test_status: pass
+    test_type: general
+    version_tested: 3.8.0
+    wcag_criterion: 1.4.3
+  - summary: Appropriate color contrast between tooltip and page background.
+    summary_additional: |
+      When you view the tooltip and use ANDI or color contrast analyzer to look at the hex colors,
+      contrast between the tooltip component itself and any adjacent background content is at least 3:1.
+    test_status: pass
+    test_type: general
+    version_tested: 3.8.0
+    wcag_criterion: 1.4.11
+# Zoom/screen magnification tests
+  - summary: Using zoom does not obstruct the tool tip or other content.
+    summary_additional: When you zoom to 200%, the tooltip content remains visible and does not overlap with other content.
+    test_status: pass
+    test_type: zoom
+    version_tested: 3.8.0
+    wcag_criterion: 1.4.4
+  - summary: Tooltip remains visible and functional when screen is magnified.
+    summary_additional: When you zoom to 200%, you can still read content in the tooltip and surrounding area easily.
+    test_status: pass
+    test_type: zoom
+    version_tested: 3.8.0
+    wcag_criterion: 1.4.10
+# Keyboard navigation tests
+  - summary: Active tooltip content can be closed with a mouse or keyboard.
+    summary_additional: When you open a tooltip with a mouse, you can use the "escape" key to close it.
```

Comment:
@amycole501 @alex-hull 
I dug into this [criteria](https://www.w3.org/TR/WCAG21/#content-on-hover-or-focus) a bit more and it looks like there are three criteria (shown below). It looks like tooltip passes "Persistent", but fails "Dismissable" and "Hoverable". Right now, the only test we have related to 1.4.13 is for "Dismissable". Is it reasonable to re-write this checklist item to include "Hoverable" and "Persistent"? If so, could we mark it as "passed with exceptions"?

> Dismissible
> A [mechanism](https://www.w3.org/TR/WCAG21/#dfn-mechanism) is available to dismiss the additional content without moving pointer hover or keyboard focus, unless the additional content communicates an [input error](https://www.w3.org/TR/WCAG21/#dfn-input-error) or does not obscure or replace other content;
>
> Hoverable
> If pointer hover can trigger the additional content, then the pointer can be moved over the additional content without the additional content disappearing;
>
> Persistent
> The additional content remains visible until the hover or focus trigger is removed, the user dismisses it, or its information is no longer valid.

Reply:
I think that is reasonable. Would it make sense to split these into three different prompts, but still have them related to the one criteria? I think that would just help it look cleaner. I also cannot remember how we have done ones with multiple considerations in the past.

Reply:
I split these out into three separate prompts in [252d4b6](https://github.com/uswds/uswds-site/pull/2646/commits/252d4b6afebb71649c5604109ea0cdd6fd6e90df). They are currently marked as "fail" but fixes are in the works and we are hoping to get those out ASAP for 3.8.1.